jdbc/src/main/java/io/kestra/jdbc/repository/AbstractJdbcExecutionRepository.java

```diff
@@ -61,11 +64,28 @@ public abstract class AbstractJdbcExecutionRepository extends AbstractJdbcReposi
     private QueueInterface<Execution> executionQueue;
     private NamespaceUtils namespaceUtils;
 
+    @Getter
+    private final JdbcFilterService filterService;
+
+    @Getter
```

Comment:
I may have missed something, but I don't think the getter is used somewhere.
In fact this should be a constant.